acvm/src/lib.rs

```diff
@@ -50,7 +48,14 @@ pub enum OpcodeResolutionError {
     #[error("unexpected opcode, expected {0}, but got {1}")]
     UnexpectedOpcode(&'static str, BlackBoxFunc),
     #[error("expected {0} inputs for function {1}, but got {2}")]
-    IncorrectNumFunctionArguments(usize, BlackBoxFunc, usize),
+    IncorrectNumFunctionArguments(usize, String, usize),
```

Comment:
why did this get changed from BlackBoxFunc to string?

Reply:
Because it is also used for the ToRadix directive.

Reply:
Perhaps we could use Opcode instead of String?

Reply:
ToRadix does not use it anymore, so I reverted to blackboxfunc.